### DISCUSSION TITLE
Thank You, homebridge-simplisafe3 contributors

### DISCUSSION THREAD

**Discussion by jparchem** [homebridge-simplisafe3/homebridge-simplisafe3#203] [General]
I've been using this plug-in now for almost a full year.  It has been rock solid stable for me arming the alarm and reporting back on the state of the various sensors I have around the home.

I upgraded to this system after SimpliSafe's CEO indicated they were going to be adding HomeKit integration with the "All-New SimpliSafe" hardware.  That never materialized but thankfully you stepped in to fill that void and allow those of us who enjoy using HomeKit to integrate Simplisafe with it.  

I hope SimpliSafe is equally appreciative of your hard work, without which, I would have since moved to another security product.  

Many thanks for all of your work on this plug-in.

**Comment by shamoon** [comment]
Hey thanks, really means a lot to hear! This project is built on many peoples work, I’m grateful for all their efforts too!

**Reply by Adam-Madet** [reply]
Awesome plugin! Keep up the good work!
